### PR TITLE
Ui branch

### PR DESCRIPTION
**Pull request by arbulgazar**

### CODE REVIEW

**Comment by vladimirslav** [comment]
Remove the picture :)

File: ui/FileEncryptor/mainwidget.cpp [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+{
+    if (!_targetFilePath.isEmpty()) {
+        ui->lineEditTargetFile->setText(_targetFilePath);
+    }
+}
+
+void MainWidget::toggleEncryptDecrypt(bool encrypt)
+{
+    if (encrypt) {
+        ui->pushButtonBrowseTargetFile->setText("Select file to encrypt");
```
Comment (vladimirslav) [inline ui/FileEncryptor/mainwidget.cpp]: Better to use static string constants for button text.

File: ui/FileEncryptor/mainwidget.cpp [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    _targetFilePath = fileDialog.getOpenFileName(this,
+                                                 tr("Select Target File"),
+                                                 startFilePath);
+    QFileInfo targetFileInfo(_targetFilePath);
+    _lastFilePath = targetFileInfo.absolutePath();
+    _targetFileName = targetFileInfo.fileName();
+
+    emit lastFilePathChanged();
+    emit targetFilePathChanged();
+    std::cout << std::endl << "Selected file path " << _targetFilePath.toStdString();
```
Comment (vladimirslav) [inline ui/FileEncryptor/mainwidget.cpp]: Where is the output displayed when the program is launched?
Better to remove it (or leave only under debug build).

**Comment by arbulgazar** [comment]
Fixed.

**Pull request merged**